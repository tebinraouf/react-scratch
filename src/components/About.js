import React from "react";

const About = () => (
	<h1>This is about us!</h1>
);

export default About;