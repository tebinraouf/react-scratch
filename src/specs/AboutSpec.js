import React from "react";
import About from "../components/About";
import renderer from "react-test-renderer";

describe("<About />", () => {
	test("Should render correctly", () => {
		const component = renderer.create(<About />);
		const tree = component.toJSON();

		expect(tree).toMatchSnapshot();
	});
});