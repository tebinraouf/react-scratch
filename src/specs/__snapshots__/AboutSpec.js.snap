// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> Should render correctly 1`] = `
<h1>
  This is about us!
</h1>
`;
